sympy/integrals/manualintegrate.py

```diff
@@ -535,14 +535,14 @@ def pull_out_u_rl(integrand):
 
         return pull_out_u_rl
 
-    liate_rules = [pull_out_u(sympy.log), pull_out_u(sympy.atan, sympy.asin, sympy.acos),
+    liate_rules = [pull_out_u(sympy.log), pull_out_u(sympy.atan, sympy.asin, sympy.acos, sympy.acot, sympy.acsc, sympy.asec),
```

Comment:
Maybe it makes sense to define a list like
```
inverse_trig_functions = [sympy.atan, ...]
```
somewhere so that it can be reused in lots of places.

Reply:
Hmm yeah that would make sense